static/css/training.css

```diff
@@ -139,20 +81,77 @@ section.call-to-action .main-section .cta-image > img {
   text-decoration: none;
 }
 
+.mainsection {
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.second {
+  padding-top:4%;
+}
+
 h5 {
   font-size: 16px;
   font-weight: 500;
   line-height: 1.5em;
   margin-bottom: 2em;
 }
 
+.logo-certification .cta-image .cta-image-after img {
+  width:300px !important;
+}
+
 .white-text {
   color: #fff;
 }
 
 .padded {
-  padding-top: 100px;
-  padding-bottom: 100px;
+  padding-top: 3%;
+  padding-bottom: 4%;
+}
+
+.padded-bottom-banner {
+  padding-top:0%;
+  padding-bottom: 2%;
+}
+
+.half {
+  width:50%;
+  float:left;
+  padding-left:10%;
+  padding-right:10%;
+  padding-top:4%;
+  padding-bottom:5%;
+}
+
+.half-image{
+  width:50%;
+  float:left;
+  display:block;
+}
+
+.half-image img{
+  width:700px;
```

Comment:
**Part 4:**

Set this to `width: 100%`. When we do mobile web stuff.

In doing so the image will fill the width of its containing div – which, in this case, is 50% of the full width of `main-section`, or ~600px. With any luck it'll scale nicely as we resize the screen without us having to do anything! 👯